### PR TITLE
Update README with information about what we've already accomplished

### PR DESCRIPTION
**Pull request by nuritzi**
Adds link to published blog post for event metrics, a talk given at GUADEC 2020, and other things that we are working on per our [wg meeting on Jan 25, 2021](https://docs.google.com/document/d/1HABrco2NGhchPLHK_PvKRsFHEU0fMfpnn0Axacv-OX8/edit#heading=h.kqagbt4r6t3m).

### CODE REVIEW

**Comment by nuritzi** [comment]
@GeorgLink can you please review? I added a link to the GUADEC talk but not sure it's appropriate. Please feel free to remove before merging.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,20 @@ A few key objectives:
...
+ * [Other identified personas](https://docs.google.com/document/d/1ghREXIO5C9PLiwXMtouOR4AjPLGz9pD34LIxO8N2-xo/edit)
+
+**In-Progress**
+ * [Promotions and Communications team persona]() -- FIXME
+ 
+**Completed**
+ * [Event organizer persona](https://github.com/chaoss/wg-app-ecosystem/blob/master/event-organizers.md) - details the metrics we've developed for the event-organizer persona and ideas on how to implement them
+   * Related blog post: [How we develop success metrics for open source events](https://opensource.com/article/20/11/chaoss-open-source-events) - an overview of this working group, the event-organizer persona, and introduction to the set of metrics we've developed for this persona
+   
+**Other outputs*
```
Comment (GeorgLink) [inline README.md:65]:
```suggestion
**Other outputs**
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
I like the structure!

I am blocked for creating the new google doc for our new persona but will add it ASAP and then we can merge all together.

File: README.md [hunk 2]
```diff
@@ -51,6 +51,20 @@ A few key objectives:
 * Build a set of metrics from those decisions
 * Present those metrics to the CHAOSS community for feedback
 
+## Reports and progress
+**Backlog**
+ * [Other identified personas](https://docs.google.com/document/d/1ghREXIO5C9PLiwXMtouOR4AjPLGz9pD34LIxO8N2-xo/edit)
+
+**In-Progress**
+ * [Promotions and Communications team persona]() -- FIXME
```
Comment (GeorgLink) [inline README.md:59]:
```suggestion
 * Persona: [Promotions and Communications Team](https://docs.google.com/document/d/12dQzl2-KWczTrNG38C9DXZg259YiIgeJDoBVV-jxU0c/edit#)
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]

**Comment by GeorgLink** [comment]
Thanks @nuritzi !

**Pull request merged**